.code-samples.meilisearch.yaml

```diff
@@ -554,6 +553,18 @@ update_sortable_attributes_1: |-
     ])
 reset_sortable_attributes_1: |-
   client.index('books').resetSortableAttributes()
+get_pagination_settings_1: |-
+update_pagination_settings_1: |-
+  client.index('books').updateSettings({ pagination: { maxTotalHits: 100 }})
+reset_pagination_settings_1: |-
+get_faceting_settings_1: |-
+update_faceting_settings_1: |-
+  client.index('books').updateSettings({ faceting: { maxValuesPerFacet: 2 }})
+reset_faceting_settings_1: |-
+settings_guide_faceting_1: |-
+  client.index('books').updateSettings({ faceting: { maxValuesPerFacet: 5 }})
```

Comment:
This method will be replaced when we implement those special methods correct?

Reply:
Yes ! Its per our discussion we had privately that i ve added them